code/modules/mob/living/carbon/human/life.dm

```diff
@@ -364,9 +364,13 @@ GLOBAL_LIST_INIT(ballmer_windows_me_msg, list("Yo man, what if, we like, uh, put
 			if(prob(25))
 				slurring += 2
 			jitteriness = max(jitteriness - 3, 0)
+			if(has_trait(TRAIT_DRUNK_HEALING))
+				adjustBruteLoss(-0.12, 0)
+				adjustFireLoss(-0.06, 0)
```

Comment:
These should probably all be `FALSE` instead of `0`, I'm not entirely sure why they're `FALSE` at all but it seems like most reagents follow the same pattern so it's probably fine.